dwave/preprocessing/cyfix_variables.pyx

```diff
@@ -49,5 +49,6 @@ def fix_variables_wrapper(bqm, strict):
         raise ValueError("bqm must be linearly indexed")
 
     cdef cyAdjVectorBQM cybqm = dimod.as_bqm(bqm, cls=AdjVectorBQM)
```

Comment:
@hhtong  This part just needs to be handled now, depending on how you return the lower bound and the fixed variables.